spec/classes/agent_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'spec_helper'
 
 describe 'sensu::agent', :type => :class do
-  on_supported_os({facterversion: '3.8.0'}).each do |os, facts|
+  on_supported_os({facterversion: '3.11.9'}).each do |os, facts|
```

Comment:
Perhaps we can do this in the spec_helper and not have to specify throughout the code?

https://github.com/mcanevet/rspec-puppet-facts#specifying-a-default-facter-version

Reply:
Change made to define facts version in one place